userspace/libsinsp/parsers.cpp

```diff
@@ -2435,45 +2436,37 @@ void sinsp_parser::parse_connect_exit(sinsp_evt *evt)
 		if(family == PPM_AF_INET6)
 		{
 			//
-			// For the moment, we only support IPv4-mapped IPv6 addresses
+			// Check to see if it's an IPv4-mapped IPv6 address
 			// (http://en.wikipedia.org/wiki/IPv6#IPv4-mapped_IPv6_addresses)
 			//
 			uint8_t* sip = packed_data + 1;
 			uint8_t* dip = packed_data + 19;
 
 			if(!(sinsp_utils::is_ipv4_mapped_ipv6(sip) && sinsp_utils::is_ipv4_mapped_ipv6(dip)))
 			{
-				evt->m_fdinfo->m_name = evt->get_param_as_str(1, &parstr, sinsp_evt::PF_SIMPLE);
```

Comment:
I suppose setting `m_fdinfo->m_name` is no longer needed here because we are properly setting the sockinfo, so the flow will be the same as v4 sockets?

Reply:
That's right. The name eventually gets set at line 2486, and both ipv6 and ipv4 follow the same steps of possibly swapping the client/server addresses.